src/core/utility/file/filePath/FilePath.cpp

```diff
@@ -238,8 +224,8 @@ std::vector<FilePath> FilePath::expandEnvironmentVariables() const {
   return paths;
 }
 
-std::vector<std::filesystem::path> FilePath::expandEnvironmentVariablesStl() const {
-  std::vector<std::filesystem::path> paths;
+std::vector<fs::path> FilePath::expandEnvironmentVariablesStl() const {
+  std::vector<fs::path> paths;
   std::string text = str();
 
   static std::regex env("\\$\\{([^}]+)\\}|%([^%]+)%");    // ${VARIABLE_NAME} or %VARIABLE_NAME%
```

Comment:
Could you move it to anonymous namespace

Reply:
I agree.